src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2435,3 +2435,174 @@ def test_log_fatal_google(self):
+    expected_security_flag = True
+    self._validate_get_crash_data(data, expected_type, expected_address,
+                                  expected_state, expected_stacktrace,
+                                  expected_security_flag)
+
+  def test_golang_panic_runtime_error_index_out_of_range(self):
+    """Test golang stacktrace with panic caused by index out of range."""
+    data = self._read_test_data(
+        'golang_panic_runtime_error_index_out_of_range.txt')
+    expected_type = 'index out of range'
+    expected_address = ''
+    expected_state = ('runtime error: index out of range\n'
+                      'http.(*conn).serve.func1\n'
+                      'panic\n')
+
+    expected_stacktrace = data
+    expected_security_flag = False
+    self._validate_get_crash_data(data, expected_type, expected_address,
+                                  expected_state, expected_stacktrace,
+                                  expected_security_flag)
+
+  def test_golang_panic_runtime_error_slice_bounds_out_of_range(self):
+    """Test golang stacktrace with panic caused by slice bounds out of range."""
+    data = self._read_test_data(
+        'golang_panic_runtime_error_slice_bounds_out_of_range.txt')
+    expected_type = 'slice bounds out of range'
+    expected_address = ''
+    expected_state = ('runtime error: slice bounds out of range\n'
+                      'pefile.(*PeFile).readImports\n'
+                      'pefile.LoadPeFile\n')
+
+    expected_stacktrace = data
+    expected_security_flag = False
+    self._validate_get_crash_data(data, expected_type, expected_address,
+                                  expected_state, expected_stacktrace,
+                                  expected_security_flag)
+
+  def test_golang_panic_runtime_error_integer_divide_by_zero(self):
+    """Test golang stacktrace with panic caused by integer divide by zero."""
+    data = self._read_test_data(
+        'golang_panic_runtime_error_integer_divide_by_zero.txt')
+    expected_type = 'integer divide by zero'
+    expected_address = ''
+    expected_state = ('runtime error: integer divide by zero\n'
+                      'go-bsbmp.(*SensorBMP180).ReadPressureMult10Pa\n'
+                      'go-bsbmp.(*BMP).ReadAltitude\n')
+
+    expected_stacktrace = data
+    expected_security_flag = False
+    self._validate_get_crash_data(data, expected_type, expected_address,
+                                  expected_state, expected_stacktrace,
+                                  expected_security_flag)
+
+  def test_golang_panic_runtime_error_makeslice_len_out_of_range(self):
+    """Test golang stacktrace with panic caused by makeslice len out of range.
+    """
+    data = self._read_test_data(
+        'golang_panic_runtime_error_makeslice_len_out_of_range.txt')
+    expected_type = 'makeslice: len out of range'
+    expected_address = ''
+    expected_state = (
+        'runtime error: makeslice: len out of range\npanic\ngc.newliveness\n')
```

Comment:
yes this runetime error frame shouldnt be in crash state, see comment above.

Reply:
Done